spec/reporting_helpers.rb

```diff
@@ -0,0 +1,40 @@
+module ReportingHelpers
+  def with_reporting_time_zones(&blk)
+    Time.use_zone(Period::REPORTING_TIME_ZONE) do
+      Groupdate.time_zone = Period::REPORTING_TIME_ZONE
+      blk.call
+      Groupdate.time_zone = nil
+    end
+  end
+
+  def test_times
+    # We explicitly set the times in the reporting TZ here, but don't use the block helper because its a hassle w/
+    # all the local vars we need
+    timezone = Time.find_zone(Period::REPORTING_TIME_ZONE)
+    now = timezone.local(2021, 6, 1, 0, 0, 0)
+    {
+      now: now,
```

Comment:
If we are using June 1 as the basis for these times, maybe change the helper to indicate that...otherwise its impossible to know what these times are when you are looking at a test in isolate. Maybe name it `june_2021`, so it would read like `june_2021[:now]` or `june_2021[:under_a_year_ago]`...

Reply:
Thanks for the naming suggestion! Fixed in e098c903.